src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -115,6 +115,35 @@ public static final class prefDrivetrain {
   }
 
   public static final class prefIntake {
+    public static final SN_DoublePreference intakePivotP = new SN_DoublePreference("ïntakePivotP", 24);
+    public static final SN_DoublePreference intakePivotI = new SN_DoublePreference("ïntakePivotI", 0);
+    public static final SN_DoublePreference intakePivotD = new SN_DoublePreference("ïntakePivotD", 0);
+    /**
+     * Enable or disable the intake pivot forward limit
+     */
+    public static final SN_BooleanPreference intakePivotForwardLimitEnable = new SN_BooleanPreference(
+        "intakePivotForwardLimitEnable", true);
+    /**
+     * <p>
+     * The maximum soft limit of the intake pivot
+     * </p>
+     * <b>Units:</b> Rotations
+     */
+    public static final SN_DoublePreference intakePivotForwardLimit = new SN_DoublePreference("intakeForwardLimit",
```

Comment:
```suggestion
    public static final SN_DoublePreference intakePivotForwardLimit = new SN_DoublePreference("intakePivotForwardLimit",
```

Reply:
ahhh I keep forgetting about the preference key 😭